genie-proto/src/main/proto/genie.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package com.netflix.genie.proto;
```

Comment:
Do we want to call it `proto` or give it a name more representative of the generated classes purposes? e.g. we have our `DTO` classes we don't call them like `JSON` or whatever.

I'm not sure what the name would be but i'm wondering if packaging them under namespace more representative of their purpose would help with clarity long term but I can also understand just saying they're proto classes

Reply:
Since they will be used in various different components, I couldn't think of a name to group them all, other than `proto`.
And keep in mind these are stubs and raw messaged, we'll need wrappers and implementations of those stubs around the code, and those can have good package names.

Reply:
Oh and the java package name is actually defined a few lines below: `com.netflix.genie.protogen`.
Any preference on one vs. the other or something else entirely?

Reply:
Changed java package name to `com.netflix.genie.proto`